Sources/JNI/SwiftJNI.swift

```diff
@@ -284,6 +308,17 @@ extension JNI {
 
         return strings
     }
+
+    public func GetObjectArrayElement(in array: JavaObjectArray, at index: Int) throws -> JavaObject {
+        let _env = self._env
+        let count = jni.GetLength(array)
+        if (index >= count) {
+            throw JNIError()
+        }
+        let jObj = _env.pointee.pointee.GetObjectArrayElement(_env, array, jsize(index))!
```

Comment:
This should not be force unwrapped with `!`. Instead, we should leave it optional, check for the JNI error (as below) and return a force-unwrapped jObj in that line